processor/src/decoder/tests.rs

```diff
@@ -1522,7 +1522,8 @@ fn set_user_op_helpers_many() {
 fn build_trace(stack_inputs: &[u64], program: &Program) -> (DecoderTrace, usize) {
     let stack_inputs = StackInputs::try_from_ints(stack_inputs.iter().copied()).unwrap();
     let mut host = DefaultHost::default();
-    host.register_handler(EMIT_EVENT_ID, Arc::new(NoopEventHandler)).unwrap();
+    host.register_handler(EventID::from_felt(EMIT_EVENT_ID), Arc::new(NoopEventHandler))
```

Comment:
nit: if instead of `EventId::from_felt()` we have `impl From<Felt> for EventId`, here, we may be able to do something like `EMIT_EVENT_ID.into()`.

Reply:
The `From` trait cannot be used in a `const` context, so we need a manual constructor.

For the others, `from_{name, u64}` I felt it was better to be explicit since we often create event IDs from constants and this makes it clear what their type is.